client/src/__tests__/TokenValidation.test.js

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import {
+  fireEvent,
+  wait,
+  render,
+  waitForElement,
+} from "@testing-library/react";
+import { act } from "react-dom/test-utils";
+
+import { ValidateToken } from "../components";
+
+// test applications use-cases from user's pov. Users access information on a web page and interact with available controls
+// assert pm react dom state
+// shallow rendering is not used for this as we need to be able to test the child components
```

Comment:
I believe you can erase this since it was put in when jest was being used.

Reply:
Fixed.